pkg/controller/data_download_controller.go

```diff
@@ -182,7 +182,7 @@ func (r *DataDownloadReconciler) Reconcile(ctx context.Context, req ctrl.Request
 
 		hostingPodLabels := map[string]string{velerov1api.DataDownloadLabel: dd.Name}
 		for _, k := range util.ThirdPartyLabels {
-			if v, err := nodeagent.GetLabelValue(ctx, r.kubeClient, dd.Namespace, k); err != nil {
+			if v, err := nodeagent.GetLabelValue(ctx, r.kubeClient, dd.Namespace, k, kube.NodeOSLinux); err != nil {
```

Comment:
By far, the ThirdPartyLabels only has one value: `"azure.workload.identity/use"`.
First, I'm not sure whether this label is only applied to Linux.
Second, I'm not sure whether Linux-specific is true for the future adding labels.

Reply:
This PR is for data mover backup only, we will have another PR for restore.
So we hardcode it to `kube.NodeOSLinux` for dataDownload.

Reply:
As for now, we don't want to support linux/Windows differential labels for the `ThirdPartyLabels` mechanism, we don't see that requirement.
In future, it is required, we come back to see how to support it.